AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml

```diff
@@ -23,5 +23,7 @@
         <element name="savePreview" type="block" selector="//button[@class='action-secondary']/span[contains(.,'Save Preview')]"/>
         <element name="imagePreview" type="block" selector="//div[@class='masonry-image-preview']//img"/>
         <element name="imagePreviewNavigation" type="block" selector="//div[@class='masonry-image-preview']//div[@class='action-buttons']/button[@class='action-{{type}}']" parameterized="true"/>
+        <element name="imageAttribute" type="block" selector="//*[@id='adobe-stock-images-search-modal']//div[@class='attributes']//span[text()='{{type}}']/parent::div//div[@class='value']//span" parameterized="true"/>
+        <element name="imageAttributeTitle" type="block" selector="//*[@id='adobe-stock-images-search-modal']//div[@class='attributes']//span[text()='{{title}}']" parameterized="true"/>
```

Comment:
My personal opinion about how selectors can be improved:
1. I don't think that @class attribute is reliable (it has a styling purpose) and should be used for selecting items in tests. I would go for data attributes (in a perfect scenario with a clear testing purpose) instead.
2. There is duplication for a modal selector. I think it's better to reuse "modal" in a way I did here: https://github.com/magento/adobe-stock-integration/blob/develop/AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAssertAdobeStockThumbnailsNumberActionGroup.xml

Reply:
ok

Reply:
2. If i have `parameterized="true"` then i not have to use this approach, is this expected be as parameter

Reply:
Let's refactor all the selectors including the modal selector in a separate pull request
`//*[@id='adobe-stock-images-search-modal']//div[@data-role='image-attributes-value']//span[text()='{{title}}']` -> `div[data-role='image-attributes-value'] span:contains('{{title}}')`